pkgs/moduleit/entrypoint.nix

```diff
@@ -11,6 +11,7 @@ let
     ];
     specialArgs = {
       inherit pkgs pkgs-23_05;
+      pkgs-unstable = pkgs;
```

Comment:
Why do we need to send pkgs in twice (once as pkgs and once as pkgs-unstable)?

Reply:
That's just so we can have in under a different name than pkgs, because I want to use pkgs internally as a local var by picking one from pkgs-unstable vs pkgs-23_05.

Reply:
Okay, yeah it looks like there is no way around this aside from renaming the pkgs inside the python module.